### PR TITLE
Removed semicolons and sorted imports

### PR DESCRIPTION
**Pull request by josipslavic**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "semi": false, "singleQuote": true }
```
Comment (anteljubic) [inline .prettierrc]: write this in multiple rows

File: src/constants/newsCategories.ts [hunk 2]
```diff
@@ -1 +1 @@
-export const newsCategories = ['worldwide', 'local', 'sport'];
+export const newsCategories = ['worldwide', 'local', 'sport']
```
Comment (anteljubic) [inline src/constants/newsCategories.ts:1]: Can you use this as an enum?

**Review by anteljubic — APPROVED** [review]
You can do the changes now or in a later pr

File: src/constants/newsCategories.ts (hunk 2 above)
Comment (josipslavic) [reply src/constants/newsCategories.ts:1]: It's being indexed in the NewsController (`newsCategories[Math.floor(Math.random() * newsCategories.length)]`) so it doesn't seem like a good idea

**Review by JCukovic — APPROVED** [review]

**Pull request merged**